test/rbs/cli_test.rb

```diff
@@ -1174,7 +1174,7 @@ def test_collection_install
           path: #{dir.join('gem_rbs_collection')}
         YAML
 
-        bundle_install('ast', 'abbrev', 'bigdecimal')
+        bundle_install('ast', 'abbrev', 'bigdecimal', 'logger')
```

Comment:
Some test cases failed due to a lack of dependencies. The `logger` package has also been moved into the bundled gem since ruby-3.5.
https://github.com/ruby/rbs/actions/runs/12742248343/job/35510085716?pr=2232